tasks/rh-sign-cosign-image/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# Install the CRDs so we can create/get them
+.github/scripts/install_crds.sh
+
+# Add RBAC so that the SA executing the tests can retrieve CRs
+kubectl apply -f .github/resources/crd_rbac.yaml
+
+# Create a dummy slack-notification-secret secret (and delete it first if it exists)
```

Comment:
`s/slack-notification/rh-sign-cosign/`

Reply:
Fixed in https://github.com/konflux-ci/release-service-catalog/pull/467.

Reply:
Thanks.